### PR TITLE
Release v1.7.5

### PR DESCRIPTION
**Pull request by bexsoft**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## Release v1.7.5
+
+Bug Fix:
+
+- Fixed leaks during ZIP multiobject downloads
+- Allow spaced in Policy names
```
Comment (bayasdev) [inline CHANGELOG.md:8]:
```suggestion
- Allow spaces in Policy names
```

**Review by bayasdev — CHANGES_REQUESTED** [review]

**Review by cesnietor — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (cesnietor) [reply CHANGELOG.md]: done @bayasdev

**Review by bayasdev — APPROVED** [review]

**Review by cesnietor — APPROVED** [review]

**Comment by bayasdev** [comment]
@bexsoft could you PTAL at the failed test? Thank you!

**Comment by bexsoft** [comment]
> @bexsoft could you PTAL at the failed test? Thank you!

This is not a real failure, if you test manually te test case passes, this is something related to testcafe & github workflows.

**Review by cniackz — APPROVED** [review]

**Pull request merged**